RELEASE.md

```diff
@@ -209,9 +209,24 @@ fill in the description. Detailed steps:
        python3 -m pip install kfp kfp-server-api --pre --upgrade
        ```
 
-       See the [Change Log](https://github.com/kubeflow/pipelines/blob/master/CHANGELOG.md)
+       See the [Change Log](https://github.com/kubeflow/pipelines/blob/$TAG_NAME/CHANGELOG.md)
        </pre>
 
+1. Update master branch to the same version.
+    ```bash
+    export TAG_NAME=<TAG_NAME>
+    git co master
```

Comment:
should we use the full command here, i.e., git checkout instead of git co. E.g., in my git version, co is not a valid command

Reply:
good point! Updated